ntbs-service/DataAccess/NtbsContext.cs

```diff
@@ -71,6 +71,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Country>(entity =>
             {
                 entity.Property(e => e.Name).HasMaxLength(200);
+                entity.HasIndex(e => e.Name);
```

Comment:
We should be indexing on `LegacyCountry`, too, as it's being used by every single query! In fact, as both these are being used in conjuction, we probably just want to have a single index on `LegacyCountry, Name`